### ISSUE TITLE
Week 46 - Wednesday - Update Wazuh feeds

### ISSUE THREAD

**Issue by Damian-Mangold** [wazuh/wazuh#26849]
|Week|Feeds|Previous issue|
|---|---|---|
|45/2024 - Wednesday|MSU/ALAS|https://github.com/wazuh/wazuh/issues/26742|
|46/2024 - Monday|NVD|https://github.com/wazuh/wazuh/issues/26800|

## Description

The following issue aims to check the weekly update of Wazuh feeds, where they will be checked, and it will be decided whether to update them according to the results obtained.

All feed results must have one of the following statuses: 
| | |
|---------------------------------|--------------------------------------------|
| :green_circle: | The feed has been successfully updated |
| :red_circle:  | The feed could not be updated due to failures |
| :yellow_circle:  | The feed has been updated, but a failure has occurred |
| :black_circle:  | The feed has not been updated because it is not needed |

Any failed feed should be reported, detailing the error and possible cause.

In the results, the differences between the current feed and the new feed that will overwrite the current feed should be detailed.

### Current Wazuh feeds
- NVD: https://feed.wazuh.com/vulnerability-detector/NVD/generated-feeds/complete_nvd_feed.json.gz
- MSU: https://feed.wazuh.com/vulnerability-detector/windows/msu-updates.json.gz
- ALAS:
  - ALAS 1: https://feed.wazuh.com/vulnerability-detector/ALAS/1/alas.json.gz
  - ALAS 2: https://feed.wazuh.com/vulnerability-detector/ALAS/2/alas2.json.gz
  - ALAS 2022: https://feed.wazuh.com/vulnerability-detector/ALAS/2022/alas2022.json.gz
  - ALAS 2023: https://feed.wazuh.com/vulnerability-detector/ALAS/2023/alas2023.json.gz

### Tasks
- [x] Check that feeds have been generated correctly.
- [x] Analyse the difference between the current feed and the new feed.
- [x] Update the current feed with the new feed.
------
- [x] In addition, check the [EoL dates](https://endoflife.date/) of the OSes we [officially support with Vulnerability Detector](https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/how-it-works.html#compatibility-matrix).
- [x] Check if there is any news related to feed deprecations on the pages of official providers, and if so, report it.

**Comment by Damian-Mangold** [comment]
#### MSU
|MSU|Hotfixes|CVEs|
|---|---|---|
| MSU | 30 | 77 |
- Hotfixes:
```
KB5002542
KB5002601
KB5002625
KB5002643
KB5002645
KB5002647
KB5002649
KB5029503
KB5033592
KB5036343
KB5037224
KB5040939
KB5042578
KB5044277
KB5044286
KB5044288
KB5044293
KB5044320
KB5044342
KB5044343
KB5044356
KB5046056
KB5046057
KB5046058
KB5046059
KB5046061
KB5046062
KB5046063
KB5046365
KB5047767
```
- CVEs:
```
CVE-2024-38203
CVE-2024-38255
CVE-2024-38264
CVE-2024-43447
CVE-2024-43449
CVE-2024-43450
CVE-2024-43451
CVE-2024-43452
CVE-2024-43459
CVE-2024-43462
CVE-2024-43530
CVE-2024-43620
CVE-2024-43621
CVE-2024-43622
CVE-2024-43623
CVE-2024-43624
CVE-2024-43625
CVE-2024-43626
CVE-2024-43627
CVE-2024-43628
CVE-2024-43629
CVE-2024-43630
CVE-2024-43631
CVE-2024-43633
CVE-2024-43634
CVE-2024-43635
CVE-2024-43636
CVE-2024-43637
CVE-2024-43638
CVE-2024-43639
CVE-2024-43640
CVE-2024-43641
CVE-2024-43642
CVE-2024-43643
CVE-2024-43644
CVE-2024-43645
CVE-2024-43646
CVE-2024-48993
CVE-2024-48994
CVE-2024-48995
CVE-2024-48996
CVE-2024-48997
CVE-2024-48998
CVE-2024-48999
CVE-2024-49000
CVE-2024-49001
CVE-2024-49002
CVE-2024-49003
CVE-2024-49004
CVE-2024-49005
CVE-2024-49006
CVE-2024-49007
CVE-2024-49008
CVE-2024-49009
CVE-2024-49010
CVE-2024-49011
CVE-2024-49012
CVE-2024-49013
CVE-2024-49014
CVE-2024-49015
CVE-2024-49016
CVE-2024-49017
CVE-2024-49018
CVE-2024-49019
CVE-2024-49021
CVE-2024-49026
CVE-2024-49027
CVE-2024-49028
CVE-2024-49029
CVE-2024-49030
CVE-2024-49031
CVE-2024-49032
CVE-2024-49033
CVE-2024-49039
CVE-2024-49040
CVE-2024-49043
CVE-2024-49046
```

####  NVD

|Action|CVEs|
|---|---|
| Update | 518 |
| Append | 448 |
---

---
### Release

| Feeds | Status | SHA256 |
|---|:-:|---|
| MSU | 🟢 | 725cc50cfb0fa57a78c9be4e53ec780be24216a217b8635a9a965a4c3d00e46d |
| NVD | 🟢 |  d21cf6287eaf87c5bac40f8bacb60df44f0da260f5e55d081cf6718c0fa3cc85 |

**Comment by Damian-Mangold** [comment]
## EoL dates

| Distribution/Provider | Versions | Maintenance Support | Extended Life Cycle Support | Comment |
|---|---|---|---|---|
|  | 5 | 31 Mar 2017 🔴 | 30 Nov 2020 🔴 | Archived ⚫ |
|  | 6 | 30 Nov 2020 🔴 | 30 Jun 2024 🔴 | Termination of support should not be a problem because they should maintain the feed - [Link](https://redhat-connect.gitbook.io/partner-guide-for-adopting-red-hat-oval-v2/faqs)|
| [**Red Hat**](https://endoflife.date/rhel) | 7 | 30 Jun 2024 🔴 | 30 Jun 2028 🟢 | |
|  | 8 | 31 May 2029 🟢 | 31 May 2032 🟢 ||
|  | 9 | 31 May 2032 🟢 | 31 May 2035 🟢 ||
|  | trusty / 14 | 02 Apr 2019 🔴 | 02 Apr 2024 🔴 | Updated - [OVAL](https://security-metadata.canonical.com/oval/) |
|  | xenial / 16 | 02 Apr 2021 🔴 | 02 Apr 2026 🟢 ||
| [**Ubuntu**](https://endoflife.date/ubuntu) | bionic / 18 | 31 May 2023 🔴 | 01 Apr 2028 🟢 ||
|  | focal / 20 | 02 Apr 2025 🟡 | 02 Apr 2030 🟢 ||
|  | jammy / 22 | 01 Apr 2027 🟢 | 09 Apr 2032 🟢 ||
|  | buster / 10 | 10 Sep 2022 🔴 | 30 Jun 2024 🔴 | #23235 |
| [**Debian**](https://endoflife.date/debian) | bullseye / 11 | 01 Jul 2024 🔴 | 30 Jun 2026 🟢 ||
|  | bookworm / 12 | 10 Jun 2026 🟢 | 10 Jun 2028 🟢 ||
|  | AL 1 | 31 Dec 2020 🔴 | 31 Dec 2023 🔴 | [Amazon announcement](https://aws.amazon.com/es/blogs/aws/update-on-amazon-linux-ami-end-of-life/) |
| [**Amazon Linux**](https://endoflife.date/amazon-linux) | AL 2 | 30 Jun 2025 🟢 | 30 Jun 2025 🟢 ||
|  | AL 2022 | - | - | Archived ⚫ |
|  | AL 2023 | 15 Mar 2025 🟡 | 15 Mar 2028 🟢 ||
| **Arch Linux** | Rolling release | - | - | 🟢 |
|  | SLES 11 & SLED 11 | 31 Mar 2019 🔴 | 31 Mar 2022 🔴 ||
| [**SUSE**](https://endoflife.date/sles) | SLES 12 & SLED 12 | 31 Oct 2024 🔴 | 31 Oct 2027 🟢 ||
|  | SLES 15 & SLED 15 | 31 Jul 2028 🟢 | 31 Jul 2031 🟢 ||
| [**AlmaLinux**](https://endoflife.date/almalinux) | 8 | 01 May 2024 🔴 | 01 Mar 2029 🟢 ||
|  | 9 | 31 May 2027 🟢 | 31 May 2032 🟢 ||
| [**Windows**](https://endoflife.date/windows) | - |  |  |  |
| [**Windows Server**](https://endoflife.date/windows-server) | - |  |  ||
| [**macOS**](https://endoflife.date/macos) | - |  |  ||

**Comment by davjo** [comment]
@Damian-Mangold , the most recent update for MSU, available at https://feed.wazuh.com/vulnerability-detector/windows/msu-updates.json.gz, was on November 13, 2024. Do you know why it hasn't been updated since then?

**Comment by Damian-Mangold** [comment]
Hello @davjo,

We would like to inform you that the vulnerability feeds have stopped updating, and there are no plans to resume their updates. The reason for this change is that, starting with Wazuh version 4.8, a refactor of the vulnerability detection module was introduced. As part of this refactor, vulnerability content is now obtained from a single source called CTI.

Therefore, to access the latest vulnerability content, it is necessary to update your environment to a Wazuh version later than 4.8. The latest available version is 4.10.0.

Here I share with you the documentation, where you can find more details of the improvements contained in the latest versions as well as a guide for the upgrade.:

 - https://documentation.wazuh.com/current/index.html

Best regards.
